Archives/ClmFile.cpp

```diff
@@ -34,27 +23,18 @@ namespace Archives
 	{
 		clmFileReader->Read((char*)&clmHeader, sizeof(ClmHeader));
 		
-		if (!clmHeader.CheckFileVersion()) {
-			throw std::runtime_error("CLM file version is incorrect.");
-		}
-
-		if (!clmHeader.CheckUnknown()) {
-			throw std::runtime_error("Unknown field in CLM header is incorrect.");
+		try {
+			clmHeader.VerifyFileVersion();
+			clmHeader.VerifyUnknown();
 		}
-
-		if (clmHeader.packedFilesCount < 1) {
-			throw std::runtime_error("Packed file count must be 1 or greater.");
+		catch (std::exception& e) {
```

Comment:
We can probably avoid using this catch. The source exceptions from the verify functions could include the CLM filename in their error message.